app/gl_js.go

```diff
@@ -32,11 +33,15 @@ func newContext(w *window) (*glContext, error) {
 	c := &glContext{
 		ctx: ctx,
 		cnv: w.cnv,
+		w:   w,
 	}
 	return c, nil
 }
 
 func (c *glContext) RenderTarget() (gpu.RenderTarget, error) {
+	if c.w.contextStatus != contextStatusOkay {
+		return nil, gpu.ErrDeviceLost
+	}
```

Comment:
That is required to prevent new frames to be generated with the old content, because was already lost.